docs/api/remote-procedure-calls-raw-transactions.md

```diff
@@ -845,14 +845,61 @@ Result:
 }
 ```
 
-_See also_
+_See also:_
 
 * [CreateRawTransaction](../api/remote-procedure-calls-raw-transactions.md#createrawtransaction): creates an unsigned serialized transaction that spends a previous output to a new output with a P2PKH or P2SH address. The transaction is not stored in the wallet or transmitted to the network.
 * [DecodeRawTransaction](../api/remote-procedure-calls-raw-transactions.md#decoderawtransaction): decodes a serialized transaction hex string into a JSON object describing the transaction.
 * [SignRawTransactionWithKey](#signrawtransactionwithkey): signs inputs for a transaction in the serialized transaction format using private keys provided in the call.
 * [SendRawTransaction](../api/remote-procedure-calls-raw-transactions.md#sendrawtransaction): validates a transaction and broadcasts it to the peer-to-peer network.
 * [Serialized Transaction Format](../reference/transactions-raw-transaction-format.md)
 
+```{eval-rst}
+.. _api-rpc-raw-transactions-getassetunlockstatuses:
+```
+
+## GetAssetUnlockStatuses
+
+The [`getassetunlockstatuses` RPC](../api/remote-procedure-calls-raw-transactions.md#getassetunlockstatuses) returns the status of the provided Asset Unlock indexes at the tip of the chain or at a particular block height if specified.
+
+_Parameters_
+
+| Name    | Type  | Presence | Description |
+| ------- | ----- | -------- | ----------- |
+| Indexes | array | Required | An array of Asset Unlock indexes (no more than 100). Each element is a numeric Asset Unlock index. |
+| height | numeric | Optional (0 or 1) | The maximum block height to check. If not specified, the chain's tip is used. |
```

Comment:
That field can be confusing. It means the field is optional and there can be either 0 instances of the field or 1. For some things (e.g. arrays) this could say "0 or more", for example. This was the way it was in the bitcoin docs we started with and we've never changed it.